var/www/html/tools/MemcachedAdmin/Library/Analysis.php

```diff
@@ -0,0 +1,332 @@
+<?php
+/**
+ * Copyright 2010 Cyrille Mahieux
```

Comment:
I was also stuck on how these were being secured, so it makes sense that you weren't thinking of keeping them.

Let's remove these now to keep it simple, but I do like having good debugging tools available (but properly secured) in production. It'd be nice to find a solution eventually.